src/MgvOfferTaking.sol

```diff
@@ -154,8 +172,8 @@ abstract contract MgvOfferTaking is MgvHasOffers {
       OfferList storage offerList;
       (sor.global, sor.local, offerList) = _config(olKey);
       mor.maxRecursionDepth = sor.global.maxRecursionDepth();
-      /* We set aside gas for ending a market order. */
-      mor.gasreqLeft = sor.local.offer_gasbase();
+      /* We have an upper limit om total gasreq for failing offers to avoid failing offers delivering nothing and exhausting gaslimit for transaction. */
```

Comment:
```suggestion
      /* We have an upper limit on total gasreq for failing offers to avoid failing offers delivering nothing and exhausting gaslimit for the transaction. */
```